lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -32,11 +32,12 @@ def name
   #
   def commands
     base = {
-      "db_connect"    => "Connect to an existing data service",
-      "db_disconnect" => "Disconnect from the current data service",
-      "db_status"     => "Show the current data service status",
-      "db_save"       => "Save the current data service connection as the default to reconnect on startup",
-      "db_remove"     => "Remove the saved data service entry"
+      "db_connect"       => "Connect to an existing data service",
+      "db_disconnect"    => "Disconnect from the current data service",
+      "db_status"        => "Show the current data service status",
+      "db_save"          => "Save the current data service connection as the default to reconnect on startup",
+      "db_remove"        => "Remove the saved data service entry",
+      "db_clear_default" => "Clears the default data service connection used on startup"
```

Comment:
These lines can be reverted now

```suggestion
      "db_connect"    => "Connect to an existing data service",
      "db_disconnect" => "Disconnect from the current data service",
      "db_status"     => "Show the current data service status",
      "db_save"       => "Save the current data service connection as the default to reconnect on startup",
      "db_remove"     => "Remove the saved data service entry"
```